docs/tutorial/AdvancedTopics.md

```diff
@@ -0,0 +1,117 @@
+# Advanced topics
+
+## Accessing state and dispatching actions from your Cmds
+
+While most of your side effect methods should be agnostic to redux, sometimes you
+will need the flexibility of accessing arbitrary slices of state or dispatching an
+action separately from the result of the cmd. Here's an example of how you can do this.
+
+```js
+import {loop, Cmd} from 'redux-loop';
+import {doSomething} from 'something.js';
+import {doSomethingResultAction} from './actions.js';
+function reducer(state, action) {
+  switch(action.type) {
+  case 'ACTION':
+    return loop(
+      {...state, initStarted: true},
+      Cmd.run(doSomething, {
+         successActionCreator: doSomethingResultAction,
+         args: [Cmd.getState, Cmd.dispatch]
+      })
+    );
+  default:
+    return state;
+  }
+}
+
+//something.js
+export function doSomething(getState, dispatch){
+   let value = getState().some.random.value;
+   dispatch(someRandomAction());
+}
+```
+
+Cmd.getState and Cmd.dispatch are just symbols that the enhanced store will
+replace with the actual functions at the time the cmd is executed. You can 
+use these symbols as parameters to your cmds as you would any other parameter (the
+order does not matter).
+
+This minimal example will cause perpetual dispatching! While it is also possible
+to make this mistake with large, complicated networks of `redux-thunk` action
+creators, it is much easier to spot the mistake before it is made. It helps to
+keep your reducers small and focused, and use `combineReducers` or manually
+compose reducers so that the number of actions you deal with at one time is
+small. A small set of actions which initiate a `loop` will help reduce the
+likelihood of causing circular dispatches.
+
+## combineReducers with redux-loop
```

Comment:
I think combineReducers should be in the tutorial and not advanced topics. It's basically required unless you make heavy use of Cmd.map or do something else creative, which someone using the tutorial probably wouldn't don

I've also had a bunch of people make issues about things not working and the problem was using vanilla combine reducers, so I think it'd be good for us to be more clear on that.

Reply:
Agreed!